biochemist-python/chapters/Working_with_Pandas.ipynb

```diff
@@ -0,0 +1,1172 @@
+       "      <td>Thrombin</td>\n",
+       "      <td>C2A</td>\n",
+       "      <td>1-(3-CHLOROPHENYL)METHANAMINE</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>9</th>\n",
+       "      <td>2C8Y</td>\n",
+       "      <td>X-RAY DIFFRACTION</td>\n",
+       "      <td>2.20</td>\n",
+       "      <td>Thrombin</td>\n",
+       "      <td>C3M</td>\n",
+       "      <td>N-[(2R,3S)-3-AMINO-2-HYDROXY-4-PHENYLBUTYL]NAP...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>10</th>\n",
+       "      <td>2C90</td>\n",
+       "      <td>X-RAY DIFFRACTION</td>\n",
+       "      <td>2.25</td>\n",
+       "      <td>Thrombin</td>\n",
+       "      <td>C1M</td>\n",
+       "      <td>1-(4-CHLOROPHENYL)-1H-TETRAZOLE</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   PDB ID             Method  Resolution        Structure Ligand ID  \\\n",
+       "0    3SHC  X-RAY DIFFRACTION        1.90   Human Thrombin       B01   \n",
+       "1    3P17  X-RAY DIFFRACTION        1.43         Thrombin       99P   \n",
+       "2    2ZNK  X-RAY DIFFRACTION        1.80         Thrombin       31U   \n",
+       "3    3SI3  X-RAY DIFFRACTION        1.55   Human Thrombin       B03   \n",
+       "4    3SI4  X-RAY DIFFRACTION        1.27   Human Thrombin       B04   \n",
+       "5    3SV2  X-RAY DIFFRACTION        1.30   Human Thrombin       P05   \n",
+       "6    1UVU  X-RAY DIFFRACTION        2.80  Bovine Thrombin       DCH   \n",
+       "7    1UVT  X-RAY DIFFRACTION        2.50  Bovine Thrombin       I48   \n",
+       "8    2C8Z  X-RAY DIFFRACTION        2.14         Thrombin       C2A   \n",
+       "9    2C8Y  X-RAY DIFFRACTION        2.20         Thrombin       C3M   \n",
+       "10   2C90  X-RAY DIFFRACTION        2.25         Thrombin       C1M   \n",
+       "\n",
+       "                                          Ligand name  \n",
+       "0   D-phenylalanyl-N-[(4-chloropyridin-2-yl)methyl...  \n",
+       "1   D-phenylalanyl-N-(pyridin-3-ylmethyl)-L-prolin...  \n",
+       "2    D-leucyl-N-(4-carbamimidoylbenzyl)-L-prolinamide  \n",
+       "3   D-phenylalanyl-N-(pyridin-2-ylmethyl)-L-prolin...  \n",
+       "4   D-phenylalanyl-N-[(1-methylpyridinium-2-yl)met...  \n",
+       "5   D-phenylalanyl-N-(pyridin-4-ylmethyl)-L-prolin...  \n",
+       "6   3-(7-DIAMINOMETHYL-NAPHTHALEN-2-YL)-PROPIONIC ...  \n",
+       "7   N-{3-METHYL-5-[2-(PYRIDIN-4-YLAMINO)-ETHOXY]-P...  \n",
+       "8                       1-(3-CHLOROPHENYL)METHANAMINE  \n",
+       "9   N-[(2R,3S)-3-AMINO-2-HYDROXY-4-PHENYLBUTYL]NAP...  \n",
+       "10                    1-(4-CHLOROPHENYL)-1H-TETRAZOLE  "
+      ]
+     },
+     "execution_count": 32,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "thrombin_df = pd.read_csv(thrombin_file) #thrombin_df is a convenient name for our dataframe.\n",
```

Comment:
I think `thrombin_file` is not on this branch.

Reply:
Done. I added thrombin_with_ligands.csv and then used os to point to it.